gui/console.cpp

```diff
@@ -582,6 +591,45 @@ void ConsoleDialog::killLastWord() {
 	}
 }
 
+void ConsoleDialog::loadHistory() {
+	Common::SaveFileManager *saveFileMan = g_system->getSavefileManager();
+	Common::InSaveFile *loadFile = saveFileMan->openRawFile("history.txt");
```

Comment:
Could we please name it `scummvm-history.txt` for lesser ambiguity

Reply:
done